### DISCUSSION TITLE
Student ID proof verification error!

### DISCUSSION THREAD

**Discussion by sleepingcat4** [community/community#17711] [GitHub Education]
Hi!

I applied for GitHub Student Benefits as a High school student last day! And I’m facing problem in terms of uploading Student ID proof! Cuz my school prints student Information on front and issue and expiration date on the back :slight_smile:

That’s why, I need to take 2 photographs in order for a successful verification of my Student ID! I’ve already opened a ticket on GitHub Support (It was closed after a few moments automatically :frowning: )

Can someone help me?

**Comment by DEEPAKKUMARPAL** [comment]
hi @sleepingcat4 , change your login mail to your school mail i.e given by your school/college
then applied for student benefits..

**Reply by sleepingcat4** [reply]
Umm! My school doesn't give us school emails and it's only for teachers :( and GitHub actually responded so thanks, man! :)

**Reply by nurullahsaeem** [reply]
> 

Did you get approved?

**Comment by nurullahsaeem** [comment]
Is there a solution of this issue?
